ricecooker/utils/downloader.py

```diff
@@ -815,6 +815,7 @@ def create_zip_dir_for_page(self, url):
 
         index_path = self.get_relative_index_path(url)
 
+        os.makedirs(os.path.dirname(os.path.join(temp_dir, index_path)))
```

Comment:
Should call `os.makedirs` with the `exist_ok=True` kwarg (https://docs.python.org/3/library/os.html#os.makedirs) so that it doesn't fail if the directory already exists.

Reply:
added!